search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -176,6 +176,7 @@
 
    ;; service related parameters
    :service-name cpv/string-plus-and-options
+   :service-type cpv/string-param-options
```

Comment:
Do we want to support the `AND` option similar to `service-name`?

Reply:
We support the following curl https://...?service_type=Harmony&service_type=OPeNDAP.  Since each service can only have 1 type it doesn't make sense to logically "and" them, you will never get any results.  We allow multiples which "or"s them together.  Is "AND"ing a logical "OR"?

Reply:
cool. Sounds like we don't have a use case for the `AND` option. Thanks.